src/clj/tasks/fetch.clj

```diff
@@ -1,36 +1,49 @@
 (ns tasks.fetch
   "NetrunnerDB import tasks"
-  (:require [web.db :refer [db] :as webdb]
+  (:require [clojure.java.io :as io]
+            [tasks.nrdb :refer [fetch-data]]
             [clojure.string :as string]
-            [clojure.java.io :as io]
-            [tasks.nrdb :refer :all]
-            [tasks.altart :refer [add-art]]))
+            [clojure.tools.cli :refer [parse-opts]]))
 
-(defn fetch-with-db
-  "Import data from NRDB. Assumes the database is already open. See `fetch` for arguments."
-  [& args]
-  (let [localpath (first (remove #(string/starts-with? % "--") args))
-        download-images (not (some #{"--no-card-images"} args))
-        data (fetch-data localpath download-images)]
-    (println (count (:cycles data)) "cycles imported")
-    (println (count (:sets data)) "sets imported")
-    (println (count (:mwls data)) "MWL versions imported")
-    (println (count (:cards data)) "cards imported")
-    (add-art false)
-    (update-config)))
+(defn usage
+  [options-summary]
+  (->> [""
+        "Usage: lein fetch [options]"
+        ""
+        "Options:"
+        options-summary]
+       (string/join \newline)))
 
-(defn fetch
-  "Import data from NetrunnerDB.
-  Can accept `--local <path>` to use the `netrunner-card-json` project locally,
-  otherwise pulls data from NRDB.
-  Specifying `--no-card-images` will not attempt to download images for cards."
-  [& args]
+(def cli-options
+  [["-l" "--local PATH" "Path to fetch card edn from"
+    :validate [#(-> %
+                    (str "/edn/raw_data.edn")
+                    io/file
+                    .exists)
+               "Could not find local data file"]]
+   ["-d" "--db" "Load card data into the database (default)"
+    :id :db
+    :default true]
+   ["-n" "--no-db" "Do not load edn data into the database"
+    :id :db
+    :parse-fn not]
+   ["-i" "--card-images" "Fetch card images from NetrunnerDB (default)"
+    :id :card-images
+    :default true]
+   ["-j" "--no-card-images" "Do not fetch card images from NetrunnerDB"
+    :id :card-images
+    :parse-fn not]
+   ])
```

Comment:
:eyes:

Reply:
I think this is ok? Unless I'm missing something.

Reply:
Just that normally we don't leave hanging parens. This isn't that big of a deal tho, so don't stress.

Reply:
ooh yes, no that's a good point! I am a fan of consistency